database/migrations/2021_04_11_172418_create_terms_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTermsTable extends Migration {
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up() {
+        Schema::create('terms', function (Blueprint $table) {
+            $table->string('id');
+            $table->primary('id');
+            $table->dateTimeTz('enter_scheduled_time');
+            $table->dateTimeTz('exit_scheduled_time');
```

Comment:
timestamp に統一しましょう（バグの温床になると嫌なので）

```suggestion
            $table->timestamp('enter_scheduled_time')->default('2000-01-01 00:00:00');
            $table->timestamp('exit_scheduled_time')->default('2000-01-01 00:00:00');
```